infrastructure/base/main.tf

```diff
@@ -8,38 +8,57 @@ terraform {
 }
 
 module "staging" {
-  source               = "./modules/env"
-  domain               = "staging.${var.domain}"
-  gcp_project_id       = var.gcp_project_id
-  gcp_region           = var.gcp_region
-  github_org           = var.github_org
-  github_project       = var.github_project
-  github_branch        = "develop"
-  google_analytics_key = var.google_analytics_key
-  google_maps_api_key  = var.google_maps_api_key
-  project_name         = var.staging_project_name
-  transifex_token      = var.transifex_token
-  sendgrid_api_key     = var.sendgrid_api_key
-  mapbox_api_key       = var.mapbox_api_key
+  source                 = "./modules/env"
+  domain                 = var.domain
```

Comment:
why drop the "staging" here?

Reply:
there's a new prefix var to account for that. we need them separated to account for the fact that the DNS zone is centralized

Reply:
OK maybe I should read till the end before asking stupid questions